x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/EqlFunctionRegistry.java

```diff
@@ -23,8 +24,9 @@ public EqlFunctionRegistry() {
         // Scalar functions
         // String
             new FunctionDefinition[] {
-                def(Substring.class, Substring::new, "substring"),
-            },
+                def(Length.class, Length::new, "length"),
+                def(Substring.class, Substring::new, "substring")
```

Comment:
honest question: what's our convention for trailing commas after the last line in a list?
just noticed the changes here and wondering which I should do in my PRs

Reply:
I think that was a leftover. I don't see a point in having a comma if there is nothing left afterwards. The compiler doesn't complain, as it's ignoring it, but imo it shouldn't be there.

Reply:
I used to Go approach with trailing comma and like the reasoning for it: 
https://dave.cheney.net/2014/10/04/that-trailing-comma